bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -130,6 +130,8 @@ Return true if a secret object should be created
     {{- true -}}
```

Comment:
In this file you must also modify the ` "external-dns.secretName"`

Reply:
That was an oversight, thanks for pointing it out. Fixed now.